R/create.R

```diff
@@ -51,7 +48,7 @@ create_package <- function(path,
     use_rstudio()
   }
   if (open) {
-    open_project(path, name, rstudio = rstudio)
+    open_project(proj_get())
```

Comment:
The previous behaviour was deliberate, so I could launch a new RStudio project from the terminal.

Reply:
I don't think I meant to change functionality 😬. You think I did? 

I was somewhat confused, though, that the `rstudio` argument of `create_*()` functions controls the addition of an Rproj file, whereas the `rstudio` argument of `open_project()` is about attempting to open in RStudio vs. changing wd of current R session.

Reply:
If you don't leave `rstudio = rstudio` here, I'm pretty sure it won't do what I want anymore.

Reply:
If I go back to a version prior to this PR and call `create_package()` in the terminal, without specifying `rstudio` at all and with specifying `rstudio = TRUE`, I don't get the new RStudio project launched either way. The new package is definitely an RStudio project, but even if I attempt to call `rstudioapi::openProject(proj_get())` explicitly I see:

``` r
> rstudioapi::openProject(proj_get())
Error: RStudio not running
```

Sorry if I am just misunderstanding what you mean by "launch a new RStudio project from the terminal".

Reply:
I often run `create_package(..., rstudio = TRUE)` in the terminal. I’m pretty sure that used to work for me. (But maybe it didn’t or it was a long time before this PR)

Reply:
Yes I think you are either misremembering or we lost that behaviour a while ago.